### ISSUE TITLE
Timeline: Update timeline entry design

### ISSUE THREAD

**Issue by htmlandbacon** [dwp/design-system#589]
## What

Explore and document design elements for timeline entry format

## Why

Following review of the [developer review of timeline](https://github.com/dwp/design-system/issues/570) we agreed we should look at the design of the headers, byline, date and body content.

## Done when

- [ ] Agreed design and component scope

## Outcomes

Once agreed we should be able to hand this ticket over for development.

## Related stories

- https://github.com/dwp/design-system/issues/570
- https://github.com/dwp/design-system/issues/590

**Comment by jonhurrell** [comment]
@claireduxs76 As discussed, we should first look to the type scale https://design-system.service.gov.uk/styles/type-scale/ to set the relationships between the elements before colour.

**Comment by claireduxs76** [comment]
After going through several design iterations, we have finalised a design to move into build phase. The design can be found on this mural board 

https://app.mural.co/t/dwpdigital7412/m/dwpdigital7412/1714053395162/6371ba06fdaea21c611de32c3c04d7baf88ce344?sender=ubb27d5a896d0c3d19d875317

Rachel Darbyshire from Access to Work is modifying her prototype to match our design and will take it into testing. I handed over the design for Colin and Simone to build. Once they have completed it, I will stress-test the prototype. I will update Confluence on the design decisions that I have made and document any further meetings or discussions with the team and the wider UCD community as we move forward.
